.github/workflows/code.yml

```
@@ -53,7 +58,7 @@ jobs:
           python-version: ${{ matrix.python }}
 
       - name: Install with latest dependencies
-        run: pip install .[dev]
+        run: .github/workflows/pip_install.sh test-${{ matrix.python }}-${{ matrix.os }} .[dev]
```

Slight preference for including the whole requirements.txt as an argument
```suggestion
        run: .github/workflows/pip_install.sh requirements-test-${{ matrix.python }}-${{ matrix.os }}.txt .[dev]
```

---

With the matrix change that would then be
```suggestion
        run: .github/workflows/pip_install.sh requirements-test-${{ matrix.python }}-${{ matrix.os }}.txt ${{ matrix.install }}
```

---

Done - this works better as the WHL requirements.txt can be done (without passing a blank argument)